WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -214,15 +215,15 @@ class PostListViewModel @Inject constructor(
         lifecycleRegistry.markState(Lifecycle.State.CREATED)
     }
 
-    fun start(site: SiteModel, targetLocalPostId: Int?) {
+    fun start(site: SiteModel, postListType: PostListType, targetLocalPostId: Int?) {
         if (isStarted) {
             return
         }
         this.site = site
         this.listDescriptor = if (site.isUsingWpComRestApi) {
-            PostListDescriptorForRestSite(site)
+            PostListDescriptorForRestSite(site, postListType.postStatuses)
```

Comment:
We may want to add the parameter names for these especially if we start utilizing more parameters. For now it's totally fine though.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9255